stable/docker-registry/templates/configmap.yaml

```diff
@@ -10,3 +10,11 @@ metadata:
 data:
   config.yml: |-
 {{ toYaml .Values.configData | indent 4 }}
+{{- if .Values.auth.token.enabled }}
```

Comment:
You can easily just add these values into your local values file.
The entire config for docker-registry  in in the values file.

Reply:
Yep, but I need the rootcertbundle as a volumeMount. For the sake of completeness I added all token auth values and set a fix rootcertbundle path

Reply:
the rootcertbundle is necessary to implement token based authentication. The registry uses the certificate to verify the token authenticity. I think this PR can be simplified. Users just need a way to mount a certificate in the container. Would you be open to a PR adding this capability?